Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:bionic
+
+ENV TZ=America/Chicago
+RUN    ln --symbolic --no-dereference --force /usr/share/zoneinfo/$TZ /etc/localtime \
+    && echo $TZ > /etc/timezone
+
+RUN apt update && apt upgrade --yes
+
+RUN apt install --yes                                                        \
+        autoconf bison build-essential clang++-6.0 clang-6.0 cmake coreutils \
+        curl diffutils flex gcc git gnupg jq libboost-test-dev libffi-dev    \
+        libgmp-dev libjemalloc-dev libmpfr-dev libstdc++6 libtool libxml2    \
+        libyaml-cpp-dev llvm-6.0 m4 make maven opam openjdk-8-jdk pandoc     \
+        pkg-config python3 python-jinja2 python-pygments python-recommonmark \
+        scala stylish-haskell time libtinfo-dev unifdef zlib1g-dev
+
+RUN update-alternatives --set java /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java
+
+RUN curl -sSL https://get.haskellstack.org/ | sh
+
+RUN    git clone 'https://github.com/z3prover/z3' --branch=z3-4.8.4 \
```

Comment:
Could we use a pre-built release of Z3?

Reply:
Well, the prebuilt installed version wasn't recent enough (at least, I thought so). Prebuilt is 4.4.0 I believe. VSC and KEVM require == 4.6.0, so we build it that way. I can test the prebuilt version for this job, see if it works.

Reply:
I'm testing the prebuilt version instead `apt install z3 libz3-dev`. I've also trimmed the list of Dockerfile `apt` dependencies.

Reply:
There are pre-built releases on GitHub, too.